modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/NullExpirationPolicy.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.runtime;
+
+import org.mule.extension.runtime.ExpirationPolicy;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Null implementation of {@link ExpirationPolicy}.
+ * <p/>
+ * This is a singleton class. Instances should be get through the {@link #INSTANCE}
+ *
+ * @since 4.0
+ */
+public final class NullExpirationPolicy implements ExpirationPolicy
+{
+
+    public static final ExpirationPolicy INSTANCE = new NullExpirationPolicy();
+
+    private NullExpirationPolicy()
+    {
+    }
+
+    /**
+     * Always returns {@code false}
+     */
+    @Override
+    public boolean isExpired(long lastUsed, TimeUnit timeUnit)
+    {
+        return false;
+    }
+
+    /**
+     * Always returns {@code 0}
+     */
+    @Override
+    public long getMaxIdleTime()
+    {
+        return 0;
```

Comment:
Maybe Integer.MAX_VALUE is better to avoid being all the time checking for expired elements